setup.py

```diff
@@ -17,9 +17,9 @@
         subp.check_call(["git", "submodule", "update"])
 
 
-# for convenience, support building extra models via extra.cfg
-# extra.cfg is not tracked by git, so can be freely modified
-# extra.cfg example:
+# for convenience, support building extra models via model.cfg
```

Comment:
It is `models.cfg`. More complete comment is in `get_models` function. Here it is better to write something like "Refer to the comments for 'get_models' on instructions for adding extra models."

Reply:
OK, changed